diagnostic_feedback/data_tool.py

```diff
@@ -0,0 +1,159 @@
+import json
+import logging
+from xblock.core import XBlock
+from xblock.fields import Scope, String, Dict, List
+from xblockutils.resources import ResourceLoader
+from .sub_api import SubmittingXBlockMixin, my_api
+from .tasks import export_dg_data as export_data_task
+loader = ResourceLoader(__name__)
+
+PAGE_SIZE = 15
+
+
+# Make '_' a no-op so we can scrape strings
+def _(text):
+    return text
+
+log = logging.getLogger(__name__)
+
+
+class ExportDataBlock(XBlock, SubmittingXBlockMixin):
```

Comment:
Same here, this is obviously based on https://github.com/open-craft/problem-builder/blob/master/problem_builder/instructor_tool.py - so rather than duplicating the code here, can we make it generic and split it out into xblock-utils or an independent data export block in its own repository?